rsk-statistics/dbQueries.js

```diff
@@ -0,0 +1,36 @@
+import format from 'pg-format';
+import dbPool from './dbPool.js';
+
+export async function getDeploymentsNumber(startDate, endDate) {
+  const queryString = format(
+    `
+    SELECT COUNT(*)
+    FROM chain_rsk_testnet.block_transactions t
+    WHERE t.signed_at >= %L
+    AND t.signed_at <= %L
+    AND t.to IS NULL
```

Comment:
This query might over-count deployments. In addition to the date range check, and that the `to` address is `0x00`, ... we should also filter out "burn" transactions. For example that the amount is more than zero and that the data is non-empty.